client/dashboard/summary-button/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+	Card,
+	CardBody,
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	Button,
+	Icon,
+} from '@wordpress/components';
+import { chevronRight } from '@wordpress/icons';
+import clsx from 'clsx';
+import { forwardRef } from 'react';
+import CoreBadge from 'calypso/components/core/badge';
+import { SummaryButtonProps, SummaryButtonFieldProps } from './types';
+import './style.scss';
+
+const noop = () => {};
+
+function SummaryButton(
+	{
+		title,
+		to,
+		decoration,
+		description,
+		strapline,
+		fields,
+		leadsToNestedPage = true,
+		onClick = noop,
+		disabled,
+		density = 'low',
+	}: SummaryButtonProps,
+	ref: React.ForwardedRef< HTMLAnchorElement >
+) {
+	const hasLowDensity = density === 'low';
+	const badges = fields?.length && (
```

Comment:
Nit, but `SummaryBadges` could be moved to a separate React component (even inside the same file)